### PR TITLE
New "Join the Community!" Page

### PR DESCRIPTION
**Pull request by smolinari**
Hope it is ok like this. :smile: 

Scott

### CODE REVIEW

File: src/guide/Join.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+## Join the Vue.js Community!
+
+The vibrant community around Vue.js is continually growing. Nevertheless, helpful hands are always welcome. In fact, a growing community is the best sign of a healthy OSS project worth putting any effort into, even if it is as the user of the software. 
+
+If you would like to also be a part of the Vue.js community, this page was meant to support your endeavors. Have a look at the resources listed below to get some orientation around the sailing ship of Vue.js and...
+
+....welcome aboard!
+
+## Resources You Should Now About!
```
Comment (azamat-sharapov) [inline src/guide/Join.md]: Know?

**Comment by qiansen1386** [comment]
Awesome!!:+1:

File: src/guide/Join.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+- [The Awesome Vue Page](https://github.com/vuejs/awesome-vue) - See what other awesome resources have been published by other awesome people.
+- [The "Show and Tell" Subforum](http://forum.vuejs.org/category/15/show-tell) - Another great place to check out what others have built with and for the growing Vue ecosystem.
+ 
+### Contributing to Vue
+
+- [The rules to contribute](https://github.com/vuejs/vue/blob/dev/CONTRIBUTING.md) - As with any community, there has to be rules, in order to ensure everyone is doing the right things the right way.
+
+- The Main Components of Vue
+  - [Vue](https://github.com/vuejs/vue) - The core repository.
+  - [Vuex](https://github.com/vuejs/vuex) - A Flux-inspired Application Architecture for VueJS.
```
Comment (yyx990803) [inline src/guide/Join.md]: I usually go with either `Vue` or `Vue.js`, never `VueJS`.

File: src/guide/Join.md (hunk 1 above)
Comment (smolinari) [inline src/guide/Join.md]: Doh! Good catch.

Scott

File: src/guide/Join.md (hunk 2 above)
Comment (smolinari) [inline src/guide/Join.md]: Yeah, I noticed that and actually thought I corrected them all. I was pretty tired when I wrote it. 

Corrections are done. :smile: 

Scott

**Comment by posva** [comment]
Nicely written @smolinari ! 👍

**Comment by smolinari** [comment]
@yyx990803 - Evan. What channels do you use inform users about new stuff? I know about Twitter. Any other means? Email? Facebook? I think any "informative" channels could also be mentioned on the page too. What do you think?

Scott

**Comment by qiansen1386** [comment]
I know there is a [announcements](http://forum.vuejs.org/category/1/announcements) section in the forum, and the [blog](http://vuejs.org/blog/) also sharing many exciting news.

**Comment by smolinari** [comment]
Ok. I can add - "Watch" the announcements forum. Are those announcements only for forum stuff though?

Blog - good point. Though, does anyone still use RSS these days? :smile: 

Scott

**Comment by qiansen1386** [comment]
I still use RSS :joy:

**Pull request merged**